docs/tutorial/getting_started_k8s.md

```diff
@@ -47,7 +47,7 @@ juju add-model demo
 Deploy the Vault K8s operator:
 
 ```shell
-juju deploy vault-k8s vault --channel=1.18/edge
+juju deploy vault-k8s vault --channel=1.18/edge --trust
```

Comment:
Do we also need to update our tf modules to use `trust`?

Reply:
Done.